### ISSUE TITLE
Add SAN to *.docker.amazee.io certificate to avoid invalid certificate issues

### ISSUE THREAD

**Issue by uandco** [pygmystack/haproxy#16]
Hi,

It seems the `*.docker.amazee.io` certificate doesn't have SAN and triggers this error, which needs to be repeatedly added as an exception:

`This server could not prove that it is project.docker.amazee.io; its security certificate does not specify Subject Alternative Names. This may be caused by a misconfiguration or an attacker intercepting your connection.`

It would be great if SAN could be added. Thanks

**Comment by Mihaic100** [comment]
Hi I second this request it would be great if SAN could be added. Thanks

**Comment by rocketeerbkw** [comment]
Pygmy has support for loading trusted local certificates which bypasses this issue of browsers not trusted the included self-signed cert https://github.com/pygmystack/pygmy#generating-local-certificates-with-mkcert.

**Comment by uandco** [comment]
> Pygmy has support for loading trusted local certificates which bypasses this issue of browsers not trusted the included self-signed cert https://github.com/pygmystack/pygmy#generating-local-certificates-with-mkcert.

Yes this was mentioned by @Mihaic100 in https://github.com/pygmystack/pygmy/issues/381 and this works, but it would still be nice to have fully trusted certs out of the box

**Comment by rocketeerbkw** [comment]
> but it would still be nice to have fully trusted certs out of the box

That's not possible, any cert we ship as an haproxy default will always be untrusted. We are more likely to improve the flow with `mkcert` since that's the better developer experience.
